cmd/lifecycle/main.go

```diff
@@ -12,6 +12,10 @@ import (
 )
 
 func main() {
+	if err := cmd.PrepareOSEnv(); err != nil {
+		cmd.Exit(err)
+	}
```

Comment:
I am not convinced we should be doing any special env logic in phases other than launch? It seems to me that we are doing something slightly unorthodox at launch time (merging `PATH` from `Config.Env` with `PATH` from registry environment) to get around a specific issue that we introduced.

Generally values in `Config.Env` are expected to override registry values in windows containers. To avoid surprising I think we should deviate from the default behavior only where necessary. Therefore if someone set `PATH` in `Config.Env` on a builder image I would expect it to override the registry value.